cmd/logger.go

```diff
@@ -17,78 +17,65 @@
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"os"
 	"path"
+	"path/filepath"
 	"runtime"
 	"strings"
+	"time"
 
-	"github.com/Sirupsen/logrus"
 	"github.com/minio/mc/pkg/console"
 	"github.com/minio/minio/pkg/errors"
 )
 
 var log = NewLogger()
 
-// LogTarget - interface for log target.
-type LogTarget interface {
-	Fire(entry *logrus.Entry) error
-	String() string
-}
+type levelType int8
 
-// BaseLogTarget - base log target.
-type BaseLogTarget struct {
-	Enable    bool `json:"enable"`
-	formatter logrus.Formatter
-}
+// Enumerated level types
+const (
+	ERROR levelType = iota // 0
```

Comment:
1. `Error` is go style of declaring enums (constant)
2. I would avoid defaulting to `ERROR` for `levelType` i.e
```go
l1 := ERROR // error level
l2 := FATAL // fatal level
var l2 levelType // l2 is not initialized but makes error level as default
```
3. no comment required about it is `// 0`

Reply:
1. Done.
2. @balamurugana : You want to avoid default value `Error`, but it is not clear to me what you want the default value to be and why `Error` is not OK. Please clarify. 
I also checked the `logrus` package code and it looks like their default value is `panic`. Do you want to have our `Fatal` level to be our default value with the same `logrus` package logic in mind? If so, I still cannot see the benefit or advantage of this suggestion.
3. Done.